dhis-2/dhis-support/dhis-support-db-migration/src/main/resources/org/hisp/dhis/db/migration/2.35/V2_35_7__Add_Uuid_User_Columns.sql

```diff
@@ -0,0 +1,7 @@
+
+
+ALTER TABLE users ADD COLUMN IF NOT EXISTS "uuid" VARCHAR(36);
```

Comment:
Check if we potentially could map to PostgreSQL `uuid` data type.

Reply:
Please check again, its now using `uuid` type instead, and using `pg-uuid` hibernate type for conversion.